platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -263,18 +273,33 @@ private MigrationReport doImport(Path exportDirectory,
         }
         report.end();
         if (report.hasErrors()) {
+            SecurityLogger.audit("Errors importing configuration settings from file {}",
+                    exportFile);
             report.record(new MigrationException(Messages.IMPORT_FAILURE, exportFile));
         } else if (report.hasWarnings()) {
+            SecurityLogger.audit("Warnings importing configuration settings from file {}",
+                    exportFile);
+            // don't leave the zip file there if the import succeeded
+            SecurityLogger.audit("Deleting exported file {}", exportFile);
```

Comment:
❓ Not a big deal, but do you think deleteQuietly would ever fail and leave the zip behind?

Reply:
If it fails then there is not much we can do. Should we audit the failure too? @stustison?

Reply:
Made changed based on our discussion